rust/sealevel/client/src/warp_route.rs

```diff
@@ -192,19 +192,25 @@ impl RouterDeployer<TokenConfig> for WarpRouteDeployer {
         app_config: &TokenConfig,
         program_id: Pubkey,
     ) {
-        if let Some(ata_payer_funding_amount) = self.ata_payer_funding_amount {
-            if matches!(
-                app_config.token_type,
-                TokenType::Collateral(_) | TokenType::Synthetic(_)
-            ) {
-                fund_ata_payer_up_to(ctx, client, program_id, ata_payer_funding_amount);
+        let try_fund_ata_payer = |ctx: &mut Context, client: &RpcClient| {
```

Comment:
Moved this because otherwise it would try to fund the ATA payer before it exists via initialization, which is fine if the amount is high enough, but I ran with a low ata payment amount which was insufficient to cover rent